components/applications-service/dev/hab-client/README.md

```diff
@@ -0,0 +1,9 @@
+# Hab Client End to End Docker Environment
+
+## Short Version
+
+1. If you don't have Automate running in the studio, enter the studio and
+   `start_all_services`
+2. In the studio, get an API token with `get_api_token`
+3. Run the script: `./run-hab-sup.sh -t YOUR_TOKEN`
```

Comment:
Do we run this inside the studio? so means that we will start a new docker container inside the docker container? 🚢 ⛵️